### DISCUSSION TITLE
No Internet Symbol on Windows 10 when using 5Ghz band, but internet works

### DISCUSSION THREAD

**Discussion by jenabaivab** [AdguardTeam/AdGuardHome#3188] [Q&A]
I run AG Home on  my windows 10 machine, and set the DNS IP on my adapter settings as 127.0.0.1.
But, if I am on the 5 Ghz band, windows shows no internet symbol when the system (or wifi) starts. 
![image](https://user-images.githubusercontent.com/60446763/119612408-7a490e00-be19-11eb-8565-562d03db23c3.png)

This issue is not present on the 2.4 Ghz band. Is this something that can be fixed?
If I switch to another dns server like cloudflare or google or something via the adapter settings, and remove the 127.0.0.1, it gets fixed too. What is the issue here ? :(

**Comment by ainar-g** [comment]
Hello and apologies for the late response.  Is the issue still there?  If so, can you [configure] AGH to collect verbose logs and post them here?  Thanks!

[configure]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog
